src/coreclr/src/jit/emitarm64.h

```diff
@@ -113,6 +113,10 @@ static UINT64 NOT_helper(UINT64 value, unsigned width);
 // A helper method to perform a bit Replicate operation
 static UINT64 Replicate_helper(UINT64 value, unsigned width, emitAttr size);
 
+// Method to do peephole optimization by checking if mov is redundant with
+// respect to the last instruction. If yes, then will omit current mov instruction.
```

Comment:
The comment here seems a bit incorrect: it implies that `IsRedundantMov` will actually alter the generated code. But that's not true: it is purely a predicate, and the caller uses it to omit a mov.

Reply:
fixed.